multi_organ_segmentation/docs/README.md

```diff
@@ -0,0 +1,111 @@
+# Multi-organ segmentation in abdominal CT
+
+### **Authors**
+
+Chen Shen<sup>1</sup>, Holger R. Roth<sup>2</sup>, Kazunari Misawa<sup>3</sup>, Kensaku Mori<sup>1</sup>
+
+1. Nagoya University, Japan
+
+2. NVIDIA Corporation, USA
+
+3. Aichi Cancer Center, Japan
+
+### **Tags**
+
+Segmentation, Multi-organ, Abdominal
+
+## **Model Description**
+
+This model uses the DiNTS model architecture searched on MSD Pancreas [1] and re-trained for multi-organ segmentation from abdominal CT images [2,3].
+
+## **Data**
+
+This model was trained on an abdominal CT dataset in portal venous phase from Aichi Cancer Center in Japan. The dataset was split into traininig, validation and testing with 300, 60 and 60 cases, respectively.
```

Comment:
Hi @cshen0625 , thanks for contributing this bundle! As for the dataset mentioned here, could you please also attach the way of how to get it (or if it is private, also need to mention here, like [this bundle does](https://github.com/Project-MONAI/model-zoo/tree/dev/models/endoscopic_tool_segmentation#data))

Reply:
Hi @yiheng-wang-nv , thank you for your comment. This dataset is a private data. I will update the README and submit it again.

Reply:
Thanks! and the file `docs/license.txt` can be removed as the dataset is not provided (like [this bundle](https://github.com/Project-MONAI/model-zoo/tree/dev/models/ventricular_short_axis_3label/docs))

Reply:
Sure. I updated the bundle based on your comments.

Reply:
Thanks! As for the bundle itself, does it have any license? Now the`LICENSE` file is empty